examples/EUVAC/json-schemas/vaccinationDisclosure.Medical.json

```diff
@@ -3,71 +3,73 @@
+            "type": "object",
+            "properties": {
+                "tg" : {"type":"string", "$comment" : "Disease or Agent Targeted." },
+                "tt" : {"type":"string", "$comment" : "Type of test." },
+                "bn" : {"type":"string", "$comment" : "Brand name." },
+                "mp" : {"type":"string", "$comment" : "Medical Product." },
+                "or" : {"type":"string", "$comment" : "Sample origin." },
+                "so" : {"type":"string", "format":"date-time", "$comment" : "Sample occurance." },
+                "ro" : {"type":"string", "format":"date-time", "$comment" : "Result occurence." },
+                "st" : {"type":"string", "$comment" : "Status." },
+                "tc" : {"type":"string", "$comment" : "Testing centre." },
+                "pi" : {"type":"string", "$comment" : "Practitioner identifier." },
+                "co" : {"type":"string", "$comment" : "Country." }
+            },
+            "required": ["tg", "tt", "bn", "mp", "or", "so", "ro", "st", "tc", "pi", "co"]
+        },
         "vaccinationEvent" : {
             "type" : "object",
             "properties" : {
-                "tg" : {"type":"string", "$comment" : "Disease or Agent Targeted" },
-                "cd" : {"type":"string", "$comment" : "Vaccine Code" },
-                "mp" : {"type":"string", "$comment" : "Medicinal Product" },
-                "ah" : {"type":"string", "$comment" : "Marketing Authorization Holder aka Manufacturer" },
-                "sq" : {"type":"string", "$comment" : "Sequence e.g. 1/2 -> First jab of 2." },
-                "lt" : {"type":"string", "$comment" : "Lot aka Batch" },
-                "oc" : {"type":"string", "format":"date", "$comment" : "Occurrence" },
-                "fr" : {"type":"string", "format":"date", "$comment" : "Valid From" },
-                "nx" : {"type":"string", "format":"date", "$comment" : "Schedule of Next event" },
-                "ap" : {"type":"string", "$comment" : "Performer Actor with function == AP ???" },
-                "pi" : {"type":"string", "$comment" : "Performer Actor - Actor Practitioner ???" },
-                "lo" : {"type":"string", "pattern" : "(-?(0|[1-9][0-9]*)(\\.[0-9]+)?([eE][+-]?[0-9]+)?),(-?(0|[1-9][0-9]*)(\\.[0-9]+)?([eE][+-]?[0-9]+)?)", "$comment" : "Location - 'latitude,longitude'." }
+                "tg" : {"type":"string", "$comment" : "Disease or Agent Targeted as codableConcept." },
+                "cd" : {"type":"string", "$comment" : "Vaccine Code as codableConcept." },
+                "mp" : {"type":"string", "$comment" : "Medicinal Product as set of codableConcepts." },
+                "ah" : {"type":"string", "$comment" : "Marketing Authorization Holder aka Manufacturer as codableConcept." },
+                "sn" : {"type":"string", "$comment" : "Series number e.g. 1 from First jab of 2." },
+                "sc" : {"type":"string", "$comment" : "Series count e.g. 2 from First jab of 2." },
```

Comment:
> from First jab of 2

the text doesn't make sense here - this will simply be a number, field is **Immunization.protocolApplied.seriesDosesPositiveInt** in FHIR Immunization spec https://www.hl7.org/fhir/immunization.html#resource

Reply:
Updated type and clarified comment.